microprofile/microprofile-config-1.2/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -1,3 +0,0 @@
-app.timeout=314159
-fileProperty=This will be overriden by a file
-yamlOrdereredProperty=Property from default config source
```

Comment:
Why is this deleted?

Reply:
unused... I fixed the typo and fixing unused typo seemed insane :)
Do you prefer empty microprofile-config.properties instead of removing it? (in config-1.2 tests)

Reply:
No, I meant all 3 properties. Not sure about `app.timeout`, but the `fileProperty` is specifically there to be sure that it is overridden by something else.

Reply:
this is used in v11... in v12 it is unused... otherwise the test would fail and I would find it in fulltext in this v12 module.. all 3 are used in org.wildfly.swarm.ts.microprofile.config.v11.HelloServlet
and unused in 
org.wildfly.swarm.ts.microprofile.config.v12.HelloServlet
Trust me in this... anyway it would/will fail in tests :)

Reply:
That is weird, I ran the entire TS once or twice, without this change, and it passed...

Reply:
I dont argue that this fixes something.. it is just unused and I ran into it when fixing typos.. no big deal

Reply:
Well I am arguing that some of these config properties are present with the intention that they will be overridden somewhere else, and so if the test observed a `This will be overriden by a file`, it would be a natural failure.

Reply:
But I'm not sure if all are like this.

Reply:
In v11 the "app.timeout" is used in servlet:
https://github.com/rhoar-qe/thorntail-test-suite/blob/123e94f997846fdcbf9c4103e2fc8665b1c19a7c/microprofile/microprofile-config-1.1/src/main/java/org/wildfly/swarm/ts/microprofile/config/v11/HelloServlet.java#L16
and verified in test:
https://github.com/rhoar-qe/thorntail-test-suite/blob/123e94f997846fdcbf9c4103e2fc8665b1c19a7c/microprofile/microprofile-config-1.1/src/test/java/org/wildfly/swarm/ts/microprofile/config/v11/MicroProfileConfig11Test.java#L22
loaded from microprofile-config.properties as expected and in v11 this stays the same (PR doesn't modify this).

In v12 the string "app.timeout" appears only in microprofile-config-1.2/src/main/resources/META-INF/microprofile-config.properties, this is not used by any annotation, parameter, not loaded and it is not tested. This is most likely just copy of microprofile-config-1.1/src/main/resources/META-INF/microprofile-config.properties from previous version (v11).

In v13 the same situation (microprofile-config-1.3/src/main/resources/META-INF/microprofile-config.properties).

This applies to all 3 properties.
This removal of "app.timeout","fileProperty","yamlOrdereredProperty" in v12 and v13 has no effect to tests because these properties were not used and affects nothing in v12 and v13.

Reply:
Ah, ok! Thanks, didn't understand it on the first try :-)